postgraphile/website/postgraphile/config.mdx

```diff
@@ -661,6 +661,17 @@ such as `role`, and will be applied by Postgres.
 
 :::
 
+:::warning
+
+Generally there's no need to set a `name` unless you have more than one pgService.
+When you do, the different services may need different connection settings and will
+definitely be listening with different "subscribers" (different events), so the `pgSettingsKey`
+and `pgSubscriberKey` dictate which key to use for this. Currently the default is if the
+name is `main` (the default name) then we use the `pgSettings`/`pgSubscriber` key, otherwise we prefix this
+with the name of the service and an underscore
```

Comment:
```suggestion
with the name of the service and an underscore, for example a pgService with
`name: "otherdb"` would use the context keys `otherdb_withPgClient`, `otherdb_pgSettings` and `otherdb_pgSubscriber`.
```

Please also integrate `withPgClient`/`withPgClientKey` into the rest of the paragraph, it will definitely change for each pgService too.